.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: go
+
+go:
+    - "1.6"
+    - "1.7"
+    - "1.8"
+    - "1.9"
+    - "1.10"
+    - "1.11"
+
+env:
+    - ROS_DOCKER=ros:kinetic-ros-base
+    - ROS_DOCKER=ros:lunar-ros-base
+    - ROS_DOCKER=ros:melodic-ros-base
+
+sudo: required
+
+services:
+    - docker
+
+install:
+    - docker build -f .travis/Dockerfile -t akio/rosgo . --build-arg ROS_DOCKER_TAG=$ROS_DOCKER
```

Comment:
`ROS_DOCKER` is used in the Dockerfile. But `ROS_DOCKER_TAG` is passed to docker build.

```
[Warning] One or more build-args [ROS_DOCKER_TAG] were not consumed
```
https://travis-ci.com/akio/rosgo/jobs/148822485#L561